### DISCUSSION TITLE
replace NeoTree with vifm and disable auto-formatting of none-ls

### DISCUSSION THREAD

**Discussion by SomeUserName1** [LazyVim/LazyVim#5120] [Q&A] [answered]
Hi,

first of all thanks for this great piece of software!

I have 2 questions:
1. How do i replace NeoTree with vifm. I'm asking as NeoTree is built into the LazyVim defaults and I don't want to break anything.
2. I'm having troubles deactivating none-ls auto-formatting. Conform is already formatting the code on save and none-ls does it again while not using the configs i set up. whats the easiest way to disable it?

Thanks in advance and kind regards
Fabian

**Comment by dpetka2001** [comment, marked as answer]
1.  You could just disable Neotree and install vifm if you like. Just like any other plugin. Just follow the docs of vifm for how to install it.
2. none-ls is not enabled by default. So you must have enable it. Disable it either through LazyExtras or if you used `import` in your `lazy.lua` file.

**Reply by SomeUserName1** [reply]
1.) it will not open in a side bar. That'll probably require some adaptation & I'm willing to do them. Just need some instructions on the how. 
If i could choose, I'd like to have the left vifm pane as replacement of NeoTree, while the right one is in the main window (configured to be the preview via vifmrc). Can you think of a way how to do that?

2.) Okay that was an easy one, thanks

**Reply by dpetka2001** [reply]
No idea about vifm. I don't use it, so can't help you with that.
